<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addCasing</class>
 <widget class="QMainWindow" name="addCasing">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>696</width>
    <height>113</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Drilling Tools - Add Casing - {well}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>70</y>
      <width>158</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="submitBtn">
       <property name="text">
        <string>Submit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelBtn">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>671</width>
      <height>48</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Hole Diameter</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Casing OD</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Casing Weight</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Casing Grade</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Casing Top</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Casing Bottom</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="holeDia"/>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="OdBox"/>
     </item>
     <item row="1" column="2">
      <widget class="QComboBox" name="weightBox"/>
     </item>
     <item row="1" column="3">
      <widget class="QComboBox" name="gradeBox"/>
     </item>
     <item row="1" column="4">
      <widget class="QLineEdit" name="casingTop"/>
     </item>
     <item row="1" column="5">
      <widget class="QLineEdit" name="casingBottom"/>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
